src/scss/layout/_header.scss

```diff
@@ -1,3 +1,68 @@
-.header {
-  height: 90px;
+@mixin  fontStyle ($color, $fontSize) {
+    color: $color;
+    font-size: $fontSize;
+    font-family: Helvetica;
+    font-style: normal;
+    font-weight: bold;
 }
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    height: 80px;
+    background: #FFFFFF;
```

Comment:
Please add all the colors as variables on the variables scss file

Reply:
please use background-color instead of background to avoid replace other background properties. And use a variable for the color

Reply:
Thanks you. Done.